_source/user-guide/alerts/configure-alerts.md

```diff
@@ -10,101 +10,166 @@ contributors:
 
-![Kibana query bar]({{site.baseurl}}/images/kibana/kibana--query-bar.png)
+![Kibana 7 search bar]({{site.baseurl}}/images/kibana/kibana-7-search-bar.png)
 
-##### Set your query in Kibana
+Refine your search query and filters,
+and review the returned logs
+to make sure you got the expected results.
 
-In Kibana, type a query in the query bar
-and press Enter.
-Review the results in the histogram and the document table,
-and make sure your query returned the expected results.
+Alerts support Lucene query language only.
+You won't be able to create an alert
+from a search written in Kibana Query Language (KQL).
+{:.info-box.important}
 
-Click **Create Alert** (to the right of the query bar).
-The _Create Alert Definition_ page is shown.
+Once you're happy with your alert,
+click **Create alert** (above the query bar).
+You're taken to the _New alert_ page.
 
 Continue with [To configure an alert](#to-configure-an-alert).
 
 #### To configure an alert {#to-configure-an-alert}
 
-![Configure an Alert]({{site.baseurl}}/images/alerts/alerts--configure-alert.png)
-
 <div class="tasklist">
 
-##### Name and tag the alert
+##### Name the alert
+
+Give your alert a meaningful name.
+
+If your alert is triggered,
+the name is used as the email subject or Slack heading.
+A helpful name gives the quick need-to-know-now information
+and doesn't use unnecessary words.
 
-Type a **Name** and a detailed **Description**.
+##### Edit the search settings
 
-Add **Tags** to help categorize this alert.
+###### Query and filters
 
-##### _(Optional)_ Edit the search settings
+![Alert query and filters]({{site.baseurl}}/images/alerts/query-and-filters.png)
 
-If you need to, change your alert **Query** and the **Accounts** that the query will search.
+If you need to change your alert query and filters,
+do that now.
 
-If you use an invalid query, the alert will be automatically disabled. Run your query in Kibana so you can be sure you're getting the expected results.
+Your query is an important source of information
+when you're investigating a triggered alert.
+Shorten your query by using filters whenever you can.
+
+If you use an invalid query,
+the alert will be automatically disabled.
+To make sure it's valid,
+test your query in Kibana.
 {:.info-box.important}
 
-##### _(Optional)_ Edit group by settings
+###### Group by
 
-![Group alert fields]({{site.baseurl}}/images/alerts/alerts--group-by.png)
+![Alert group by settings]({{site.baseurl}}/images/alerts/alerts--group-by.png)
 
-Click **Add group by** to add up to 3 groups.
+Moving down, you'll come to the **Group by** setting next.
 
-In the **Select Field** list,
-choose a field to group by.
+With _Group by_, you can choose up to 3 fields to group.
+The alert will return the aggregated results for each group.
 
-To limit the available fields,
-choose a log type from the **Filter by type** list.
-To show fields for all log types, choose **Clear filter**.
+Careful, though: The order of these fields matters.
```

Comment:
Classic use-case of a warning box :D

Reply:
Also - group by order has NO effect on results. so this whole section is puzzling to me.

Reply:
@AmiStrn That's contrary to everything I've been told so far. Can you check and confirm that please?

Reply:
@AmiStrn In the old page, the group-by had an effect on the results.

Reply:
Do we have an answer on this?
@idohalevi @AmiStrn @AmirTugi @schwin007

Reply:
> Classic use-case of a warning box :D

Wasn't going to, but now that I can take out the one above it, I will!

Reply:
Just received word that order does indeed matter. The text will remain.